graphql/database/experiments/features.js

```diff
@@ -198,6 +199,47 @@ const features = {
       },
     ],
   },
+  [SFAC_EXTENSION_PROMPT]: {
+    defaultValue: 'Control',
+    rules: [
+      /* Begin internal overrides */
+      {
+        condition: {
+          v4BetaEnabled: {
+            $eq: true,
```

Comment:
Add the same "tabs" condition to both of these overrides so our internal testing reflects actual behavior.